core/src/components/toggle/toggle.tsx

```diff
@@ -115,6 +116,13 @@ export class Toggle implements ComponentInterface {
    */
   @Prop() justify: 'start' | 'end' | 'space-between' = 'space-between';
 
+  /**
+   * How to pack the label and control along the cross axis.
```

Comment:
Can we update this comment (and ditto for other components) that this only applies when the label and control are on separate lines such as when using `labelPlacement="stacked"`?